test/functional/tests/io_class/test_io_class_cli.py

```diff
@@ -0,0 +1,104 @@
+        csv = casadm.list_io_classes(cache.cache_id, OutputFormat.csv).stdout
+        if not IoClass.compare_ioclass_lists(IoClass.csv_to_list(csv), default_list):
+            TestRun.LOGGER.error("Default configuration does not match expected\n"
+                                 f"Current:\n{csv}\n"
+                                 f"Expected:{IoClass.list_to_csv(default_list)}")
+
+    with TestRun.LOGGER.step("Create and load configuration file for 33 IO classes "
+                             "with random names, allocation and priority values"):
+        random_list = IoClass.generate_random_ioclass_list(33)
+        IoClass.save_list_to_config_file(random_list, ioclass_config_path=ioclass_config_path)
+        casadm.load_io_classes(cache.cache_id, ioclass_config_path)
+
+    with TestRun.LOGGER.step("Display and export IO class configuration - displayed configuration "
+                             "should be the same as created"):
+        TestRun.executor.run(
+            f"{casadm.list_io_classes_cmd(str(cache.cache_id), OutputFormat.csv.name)}" 
+            f" > {saved_config_path}")
+        csv = fs_utils.read_file(saved_config_path)
+        if not IoClass.compare_ioclass_lists(IoClass.csv_to_list(csv), random_list):
+            TestRun.LOGGER.error("Exported configuration does not match expected\n"
+                                 f"Current:\n{csv}\n"
+                                 f"Expected:{IoClass.list_to_csv(random_list)}")
+
+    with TestRun.LOGGER.step("Stop Intel CAS"):
+        casadm.stop_cache(cache.cache_id)
+
+    with TestRun.LOGGER.step("Start cache and add core"):
+        cache = casadm.start_cache(cache.cache_device, force=True)
+        casadm.add_core(cache, core.core_device)
+
+    with TestRun.LOGGER.step("Check IO class configuration (should contain only default class)"):
+        csv = casadm.list_io_classes(cache.cache_id, OutputFormat.csv).stdout
+        if not IoClass.compare_ioclass_lists(IoClass.csv_to_list(csv), default_list):
+            TestRun.LOGGER.error("Default configuration does not match expected\n"
+                                 f"Current:\n{csv}\n"
+                                 f"Expected:{IoClass.list_to_csv(default_list)}")
+
+    with TestRun.LOGGER.step("Load exported configuration file for 33 IO classes"):
+        casadm.load_io_classes(cache.cache_id, saved_config_path)
+
+    with TestRun.LOGGER.step("Display IO class configuration - should be the same as created"):
+        csv = casadm.list_io_classes(cache.cache_id, OutputFormat.csv).stdout
+        if not IoClass.compare_ioclass_lists(IoClass.csv_to_list(csv), random_list):
+            TestRun.LOGGER.error("Exported configuration does not match expected\n"
+                                 f"Current:\n{csv}\n"
+                                 f"Expected:{IoClass.list_to_csv(random_list)}")
+
+    fs_utils.remove(saved_config_path)
```

Comment:
This should be in separate step, e.g. "Cleanup".

Reply:
As above